laviii123/Btecky
# Fixes- Snake Game Using Html CSS JS

I am adding the snake game by using HTML, CSS, JS

Fixes #274 

Video-

https://github.com/laviii123/Btecky/assets/91898349/b9f7f3de-9509-4bfd-8d48-d4b488de6f19


Type of change
1. Add Music with Every movement of the game
2. That Game show real-time score and High score

How should this be tested?
1. Go to the IDE 
3. Open the folder Snake_Game
4. Then check it out all file 
5. Enjoy the Game
